components/motor/tmcstepper/stepper_motor_tmc.go

```diff
@@ -112,7 +115,7 @@ const (
 	uSteps  = 256      // Microsteps per fullstep
 )
 
-// TMC5072 Register Addressses (for motor index 0)
+// TMC5072 Register Addressses (for motor index 1)
```

Comment:
Can't comment on the line, but could we unexport `NewMotor` on line 154, while we're at it?